packages/loader/driver-utils/src/runWithRetry.ts

```diff
@@ -84,6 +84,15 @@ export async function runWithRetry<T>(
                 );
             }
 
+            // logging the first failed retry
+            if(numRetries === 0) {
+                logger.sendTelemetryEvent({
+                    eventName: `${fetchCallName}_firstFailed`,
```

Comment:
do we need to differentiate between the two with an event name prefix or something like that? Or is the `callName` enough?

Reply:
no need for that, the one in odsp will have a ODSP driver prefix in eventName